### ISSUE TITLE
Player doesn't display ID tags when playing online radio stations

### ISSUE THREAD

**Issue by ptkovsky** [losnoco/Cog#15]
Tags are not visible after adding a playlist file with the list of online radio stations. Some of the entries are not even detected properly. For example, player reads http://127.0.0.1:8000/stream.mp3 but it somehow dislikes another playlist entry like http://127.0.0.1:8000/. In the latter case it just shows / symbol.

**Comment by kode54** [comment]
As it is right now, streamed and remote HTTP files don't support tags at all. Streaming support is really limited right now. Adding support for reading tags from a live stream is a welcome addition, if anyone wants to help implement that. It wasn't exactly there when I first forked the project, either.

**Comment by megadr01d** [comment]
@kode54 In the mean time, would it be feasible to just implement title reading from `m3u` and `pls` files?

`pls`

```ini
[playlist]
Version=2
NumberOfEntries=3
File1=http://radio.plaza.one/ogg
Title1=Nightwave Plaza [Vorbis 96kbps]
Length1=-1
File2=http://radio.plaza.one/mp3
Title2=Nightwave Plaza [MP3 128kbps]
Length2=-1
File3=http://radio.plaza.one/opus
Title3=Nightwave Plaza [Opus 96kbps]
Length3=-1
```

`m3u`

```ini
#EXTM3U

#EXTINF:0,Nightwave Plaza [Vorbis 96kbps]
http://radio.plaza.one/ogg

#EXTINF:0,Nightwave Plaza [MP3 128kbps]
http://radio.plaza.one/mp3

#EXTINF:0,Nightwave Plaza [Opus 96kbps]
http://radio.plaza.one/opus
```

With the example playlists above, Cog would show

```
Nightwave Plaza [Vorbis 96kbps]
Nightwave Plaza [MP3 128kbps]
Nightwave Plaza [Opus 96kbps]
```

instead of just

```
ogg
mp3
opus
```

**Comment by maya923** [comment]
> As it is right now, streamed and remote HTTP files don't support tags at all. Streaming support is really limited right now. Adding support for reading tags from a live stream is a welcome addition, if anyone wants to help implement that. It wasn't exactly there when I first forked the project, either.

If this is the case, how does Shoutcast and Icecast streams work when people listen in iTunes? Even in the macOS version of DeaDBeeF, the tags for streamed MP3s show nicely, @ least with this station it does:  http://sleepbot.com/ambience/broadcast/

**Comment by kode54** [comment]
The system certainly doesn't implement it for you. I should have mentioned, this player has zero support for dynamic tags, whatsoever. What the input reads is added to the playlist when you add the file, and the tags are never checked again.

**Comment by kode54** [comment]
Fixed by 0012d1b17e1e6e7b9cd86be0e1317f1a3949ab8c
